src/tokenizers/config.rs

```diff
@@ -0,0 +1,47 @@
+use crate::error::BoxedError;
+use std::path::PathBuf;
+
+use snafu::Snafu;
```

Comment:
Import order, we should use: standard library imports, blank line, third-party crate imports, blank line, local imports. So in this case:

```suggestion
use std::path::PathBuf;

use snafu::Snafu;

use crate::error::BoxedError;
```

Reply:
Huh, I thought `rustfmt` handled import organization automatically?

Reply:
Only within a group. What we want is `StdExternalCrate` for the `group_imports` setting:

https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#group_imports

But it is only available in `unstable`. Though a sequence of unstable `cargo fmt` with this setting, followed by stable `cargo fmt` should do the trick.